CONTRIBUTING.md

```diff
@@ -0,0 +1,13 @@
+# Contributing
```

Comment:
I think one should keep the lists sorted alphabetically while adding a project.

Need your suggestions on this